gef.py

```diff
@@ -1357,15 +1361,15 @@ def __iter__(self):
             current_chunk = next_chunk
         return
 
-    def get_next_chunk(self, allow_unaligned=False):
+    def get_next_chunk(self, allow_unaligned: bool = False):
```

Comment:
return type?

Reply:
not possible to reference the type `GlibcChunk` inside the class definition of `GlibcChunk` with py3.6